rfcs/2022-05-24-boolean-ops.md

```diff
@@ -0,0 +1,18 @@
+- Feature Name: `boolean-ops`
+- Start Date: 2022-05-24
+- Feature PR: //github.com/georust/geo/pull/835
```

Comment:
Just a note: the below links seem to be rendered correctly, but github's markdown renderer doesn't like this link: 
<img width="304" alt="image" src="https://user-images.githubusercontent.com/15164633/173126559-43728a3c-8e11-4e07-9b41-5a4348e09756.png">